app/views/claims/new.html.erb

```diff
@@ -9,8 +9,7 @@
         
     <%= simple_form_for :claim, url: contribution_claims_path(contribution) do |f| %>
       <%= f.input :peer_alias, as: :string, label: false, placeholder: "Alias or handle (do not use your real name here)", required: true %>
-      <%= f.input :preferred_contact_method_id, collection: Array(ContactMethod.email), label: "Preferred contact method", required: true %>
-      <%= f.input :preferred_contact_info, as: :string, placeholder: "Add your phone number or email address", input_html: { value: preferred_contact_info }, required: true  %>
+      <%= f.input :email, as: :string, placeholder: "Add your phone number or email address", input_html: { value: preferred_contact_info }, required: true  %>
```

Comment:
The placeholder still mentions phone number. We could probably drop the placeholder TBH.
Also, is the reference to `.preferred_contact_info` still necessary?

Reply:
That's a good catch. I've cleaned this up and also made changes to the `new` action in controller.